.github/workflows/ci.yaml

```diff
@@ -0,0 +1,472 @@
+name: CI
+
+on: [ push ]
```

Comment:
Isn't it too much to run on `push`? (should it be `pull_request`?)

Reply:
Yes, I could limit it to `pull_request` event. Let me confirm with Toshi.

@brfrn169 The current circle-ci config triggers on each `push` event. Do you see any problem limiting it to the `pull_request` event?

Reply:
My only concern is cost, so if the cost doesn't increase linearly for the number of pushes, then it should be fine. (Github Actions cost is supposed to be free for public repositories, but I'm not fully sure about the other costs for the CIs)

So, I'm fine with `push` if it doesn't make much difference in cost and provides more development productivity.
If it costs too much, the alternative would be `pull_request` and `workflow_dispatch` so that we can still run the CIs when needed.

Reply:
In most of my cases, the timing of `push` is the timing where I want to run the CI. However, I'm not sure the same is true for others. So cost-wise, it's safe that the triggers of the CI are `pull_request` and `workflow_dispatch`.

@Torch3333 Can you please change the the triggers of the CI to `pull_request` and `workflow_dispatch`?

Reply:
Sure